src/test/cuckoocache_tests.cpp

```diff
@@ -163,7 +163,7 @@ void test_cache_erase(size_t megabytes)
     for (uint32_t i = (n_insert / 2); i < n_insert; ++i)
         set.insert(hashes_insert_copy[i]);
 
-    /** elements that we marked erased but that are still there */
+    /** elements that we marked erased but are still there */
```

Comment:
Nit: Also change "marked erased" to "marked as erased".

Reply:
Fixed.